### ISSUE TITLE
Улучшения UI при поиске по Vin

### ISSUE THREAD

**Issue by makovsv** [makovpv/Carzine#69]
1. При введенном вин не надо над строкой поиска писать фразу "Искать запчасти по Вин или номеру детали"

2. В предлагаемой модели автомобиля для поиска, при указании года не надо писать букву г

3. Саму модель указывать как ссылку чтобы было понятно что она кликабельна

4. Указывать последние три машины из гаража

5. При успешном поиске по вин, добавить заголовок после марки модели: "Выберите запчасть:" и далее уже вниз идут категории

6. В категории запчастей не использовать символ *

![Image](https://github.com/makovpv/Carzine/assets/38849646/b95e3986-9c9e-49c3-8442-da6d8343096a)

**Comment by makovpv** [comment]
>В категории запчастей не использовать символ *

Символ * я добавляю для тех групп, которые содержат подгруппы. Давай уберу, но мне кажется надо как-то давать понять пользователю, что у группы есть подгруппы

**Comment by makovsv** [comment]
Да, предлагаю убрать, так как пользователь его все равно не считывает нужным нам образом
